### PR TITLE
Switch to setup-edgedb and touchup the release workflow

### PR DESCRIPTION
**Pull request by elprans**

### CODE REVIEW

File: edgedb/_cluster.py [hunk 1]
```diff
@@ -95,10 +95,7 @@ def __init__(
 
     @staticmethod
     def get_edgedb_server():
-        try:
-            return f'edgedb-server-{os.environ["EDGEDB_CI_SLOT"]}'
-        except KeyError:
-            return 'edgedb-server'
+        return 'edgedb-server'
```
Comment (tailhook) [inline edgedb/_cluster.py:98]: What's changed here? We still need binary with a suffix.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9]
         os: [ubuntu-latest, macos-latest]
```
Comment (tailhook) [inline .github/workflows/tests.yml:19]: Can we also add the following?
```
edgedb-version: [stable, nightly]
```

File: edgedb/_cluster.py (hunk 1 above)
Comment (elprans) [reply edgedb/_cluster.py:98]: No, `edgedb-setup` adds the directory with unsuffixed `edgedb-server` to `PATH`.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (elprans) [reply .github/workflows/tests.yml:20]: Good idea.

**Review by 1st1 — APPROVED** [review]

**Comment by tailhook** [comment]
Why this wasn't merged yet? I want this merged to update edgedb tests run with. Can we merge this, while marking #160 as expected failure.

**Comment by elprans** [comment]
I forgot this wasn't merged.

**Pull request merged**